cohortRouter.js

```diff
@@ -0,0 +1,133 @@
+const express = require("express");
+const knex = require("knex");
+
+const knexConfig = require("./knexfile");
```

Comment:
you can avoid having to establish your db like this if instead you export knexConfig.development from your knexfile instead